src/dedicated/overview/backups.md

```diff
@@ -0,0 +1,22 @@
+# Backups & snapshots
```

Comment:
I think we got rid of referring to this as snapshots > backups, unless that's not the cast on Dedicated.

Reply:
I don't think they were ever called snapshots on Ent/Dedicated...

Reply:
```suggestion
# Backups
```